duckduckgo/iOS
# Improved Broken sites process

Task/Issue URL: https://app.asana.com/0/856498667320406/1154001374942564
Tech Design URL: https://app.asana.com/0/72649045549333/1163624025952315
CC:

**Description**:
Update UI to support picking up category. Send a pixel with the broken site data as per the extensions.

**Steps to test this PR**:

Broken site form:
- Submit button should be disabled on start.
- Category entries should be randomized.
- Dark/Light theme should be supported.
- On iPhone: navigating from Privacy Screen should open modal view stacked with Privacy Screen.
- On iPhone: navigating from Menu should open full screen modal view.
- On iPad: both navigation paths should open modal view (non-full screen one).

Sending report should include:
- TDS etag (even for embedded TDS if falling back to it).
- Loaded surrogates.
- Domains of blocked trackers.
- Mobile/Desktop and HTTPS upgrade status.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
